app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/ContractDetailDestination.kt

```diff
@@ -127,14 +140,23 @@ private fun ContractDetailScreen(
         }
       },
       label = "contract detail screen fade animated content",
-      modifier = Modifier.weight(1f),
+      transitionSpec = { fadeIn() togetherWith fadeOut() },
```

Comment:
I think we still want the weight here so that the layout still always tries to take up the entire height. I remember the problem was that for example if you are only in the documents tab, which is quite short and often does not have enough content to cover the entire screen, then going back to the Coverage tab you'd see clipping for a bit until it animated back up again.